frontend/src/pages/hardwareProfiles/ManageNodeResourceSection.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FormSection, Flex, FlexItem, Button } from '@patternfly/react-core';
+import { Identifier } from '~/types';
+import NodeResourceTable from './nodeResource/NodeResourceTable';
+import ManageNodeResourceModal from './nodeResource/ManageNodeResourceModal';
+
+type ManageNodeResourceSectionProps = {
+  nodeResources: Identifier[];
+  setNodeResources: (identifiers: Identifier[]) => void;
+};
+
+export const ManageNodeResourceSection: React.FC<ManageNodeResourceSectionProps> = ({
+  nodeResources,
+  setNodeResources,
+}) => {
+  const [isNodeResourceModalOpen, setIsNodeResourceModalOpen] = React.useState<boolean>(false);
+  return (
+    <>
+      <FormSection
```

Comment:
In the mockup, there is a description under this section title. Also, the button is a link style below the table but not next to the section title. However, the description is missed here and the button becomes the secondary button next to the title. Can you help confirm that @vconzola?

<img width="700" alt="Screenshot 2024-12-18 at 5 16 54 PM" src="https://github.com/user-attachments/assets/30f8ec56-c4b0-497f-9386-25d77fe3be9d" />

Reply:
Please implement as shown in the mockup including the description, the action icons in place of a kebab and the link button for Add resource. Thanks.

Reply:
@dpanshug Please refer to the implementation [here](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/BYONImages/BYONImageModal/DisplayedContentTableRow.tsx#L86-L133)

Reply:
this is not the latest mocks. did you verify with the latest

Reply:
Where is the latest mock? I checked this one https://www.figma.com/design/UFhLtsPxoETwd39pnMsASV/Hardware-profiles?node-id=0-1&p=f&t=52XPdoeFVdZ2Bkl6-0.

Reply:
@vconzola For the button we have discussed before [here](https://redhat-internal.slack.com/archives/C07C0FNHVPT/p1733934994680699?thread_ts=1733348827.112909&cid=C07C0FNHVPT). Can you confirm if it should be in table toolbar or the bottom of the table when the table is not empty?